examples/simplicial_2d_dg/elixir_euler_sbp_triangular_mesh_flux_diff.jl

```diff
@@ -0,0 +1,53 @@
+# !!! warning "Experimental features"
+
+using StartUpDG
+using Trixi, OrdinaryDiffEq
+
+polydeg = 3
+rd = RefElemData(Tri(), SBP(), polydeg)
+
+volume_flux = flux_ranocha
+dg = DG(rd, nothing #= mortar =#,
+        SurfaceIntegralWeakForm(FluxLaxFriedrichs()), VolumeIntegralFluxDifferencing(volume_flux))
```

Comment:
Did you check whether you get semidiscrete EC when setting
```julia
volume_flux = flux_ranocha
dg = DG(rd, nothing #= mortar =#,
        SurfaceIntegralWeakForm(surface_flux=flux_ranocha), VolumeIntegralFluxDifferencing(volume_flux))
```
You could use the same setup that we use for other EC tests for the Euler equations, i.e., `initial_condition_weak_blast_wave` and no source terms (see `examples/tree_2d_dgsem/elixir_euler_ec.jl`).

Reply:
I'll add a check soon - thanks!

Reply:
Is there a template for how to report results from the EC test? If not, I'll just compute the contraction of entropy variables and RHS and test that it's less than some tolerance.

Reply:
The analysis callback is constructed using the `default_analysis_integrals` at https://github.com/trixi-framework/Trixi.jl/blob/83bd625bc335c75ed99a905f31d022231ca59c8c/src/equations/equations.jl#L197
We have a dispatch on that, e.g. at https://github.com/trixi-framework/Trixi.jl/blob/83bd625bc335c75ed99a905f31d022231ca59c8c/src/callbacks_step/analysis_dg2d.jl#L190-L199
It would be great to get something similar for your new meshes/solvers

Reply:
Oh, right - I forgot I implemented this previously. Thanks for the reminder!

Reply:
I can verify that the entropy time derivative is zero to machine precision over the entire simulation from the analysis printout. How should one extract the entropy time derivative for a test though? There doesn't seem to be a clean interface for this.

Reply:
No, there isn't.... But it's great that you checked it :+1:

Reply:
If that's the case, I'll leave the elixir where it is for now then (`elixir_euler_periodic_triangular_mesh_ec.jl`), but won't add it to the tests yet. 

It might be nice to have something like `TimeSeriesCallback` which stores global integrated quantities over time (returning a vector of `t_values` and `integral_values_at_t` or something). If there's no current feature like that, I can open an issue (and try to take a stab when I have time).

Reply:
Leaving the elixir as it is is fine :+1:
We have discussed to extend the timeseries callback this way. Right now, you need the analysis callback if you want to use integrated quantities. You can save the analysis output to a file and read it in later. I don't remember where exactly the discussion happened, but I hope we have an issue for that :sweat_smile: If not, please open an issue :+1:

Reply:
Looks like https://github.com/trixi-framework/Trixi.jl/issues/589.

Reply:
For now, you can use `save_analysis=true` if you want to access the quantities calculated by the `AnalysisCallback` in a simple text file:
https://github.com/trixi-framework/Trixi.jl/blob/50ba5ffb0dd9f8933b916c2271c7bcbc22122723/examples/tree_2d_dgsem/elixir_euler_vortex.jl#L31-L34

Reply:
Also, it would *really* be great if you can take a stab at #589!! @ranocha has already some ideas about how to implement it, so it would make sense to coordinate with him as well before you start.

Reply:
Let's resolve this conversation here. Further communication can be done in #589.